UI/platform-windows.cpp

```diff
@@ -499,6 +499,58 @@ void TaskbarOverlaySetStatus(TaskbarOverlayStatus status)
 }
 #else
 // Needs to be re-implemented for Qt6, perhaps natively without Qt
```

Comment:
This comment can be removed with your changes since you did implement it.

Reply:
Ah thanks for that